### PR TITLE
Fix Windows build, add run_exports and add traversaro as a mantainer

### LINKED ISSUE #10: ImportError/build error when using portaudio with pyaudio on Windows x64
I filed this issue previously at [pyaudio-feedstock](https://github.com/conda-forge/pyaudio-feedstock/issues/4).
Short summary: `portaudio` provided by `conda-forge` does not work for me on Windows (ImportError) and I can also not use it to build `pyaudio` (so far) due to missing headers.

### Usage issue
As there is no conda-forge version of `pyaudio`, `conda` tries to install 

```
  portaudio          conda-forge/win-64::portaudio-19.6.0-h6538335_2
  pyaudio            pkgs/main/win-64::pyaudio-0.2.11-py37hfa6e2cd_1
```

This installation causes and `ImportError` due to DLL loading failures:

```
>>> import pyaudio
...
# trying C:\Miniconda3-x64\envs\test-environment\lib\site-packages\_portaudio.cp37-win_amd64.pyd
Could not import the PyAudio C module '_portaudio'.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Miniconda3-x64\envs\test-environment\lib\site-packages\pyaudio.py", line 116, in <module>
    import _portaudio as pa
ImportError: DLL load failed: The specified module could not be found.
```

If I manually override portaudio with `portaudio pkgs/main/win-64::portaudio-19.6.0-hfa6e2cd_3` `pyaudio` works (at least I can import it).

### Build Issue
Now @ocefpaf suggested to add Windows support to the `conda-forge` recipe of `pyaudio`. I gave that a try and stumbled upon missing headers. The official `portaudio` package seems to install headers (e.g portaudio.h) into `%CONDA%\envs\%ENV%\Library\include`. I could not find these headers when installing `portaudio` from `conda-forge`.

Do you have any suggestions about how to make `pyaudio` work with `portaudio` on Windows? How do the `portaudio` packages in `pkgs/main` and `conda-forge` differ? What needs to be done to allow `pyaudio` to build with `portaudio` on Windows?

### PR DESCRIPTION
**Pull request by traversaro**
This should fix https://github.com/conda-forge/portaudio-feedstock/issues/10 .

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by traversaro** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,42 @@
...
   run:
     - alsa-lib  # [linux]
 
 test:
   commands:
     - test -f $PREFIX/lib/libportaudio$SHLIB_EXT  # [not win]
+    - test -f $PREFIX/include/portaudio.h  # [not win]
+    - if not exist %PREFIX%\\Library\\include\\portaudio.h exit 1  # [win]
+    - if not exist $PREFIX$\\Library\\lib\\portaudio.lib exit 1  # [win]
+    - if not exist $PREFIX$\\Library\\bin\\portaudio.dll exit 1  # [win]
```
Comment (ryanvolz) [inline recipe/meta.yaml]:
```suggestion
    - if not exist %PREFIX%\\Library\\lib\\portaudio.lib exit 1  # [win]
    - if not exist %PREFIX%\\Library\\bin\\portaudio.dll exit 1  # [win]
```

File: recipe/bld.bat [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+      -DTARGET_POSTFIX= ^
+      .
 if errorlevel 1 exit 1
 
-exit 0
+cmake --build . --target ALL_BUILD --config Release
+if errorlevel 1 exit 1
+
+copy /y Release\*.dll %LIBRARY_BIN%
+copy /y Release\*.lib %LIBRARY_LIB%
```
Comment (ryanvolz) [inline recipe/bld.bat]: Would be good to not include the `portaudio_static.lib` in the package per [CFEP-18](https://github.com/conda-forge/cfep/blob/master/cfep-18.md).

**Review by ryanvolz — COMMENTED** [review]
I just stumbled across this because I noticed that the existing `portaudio` Windows package was not installing to the right location in the prefix and has those pesky MinGW names.

Thanks for already fixing these things (mostly)!

File: recipe/bld.bat (hunk 2 above)
Comment (traversaro) [reply recipe/bld.bat]: Fixed.

**Comment by traversaro** [comment]
Thanks for the feedback @ryanvolz !

**Comment by ocefpaf** [comment]
@traversaro I don't know the first thing about Windows. If you tell me this is good to go I'll merge it.

**Comment by traversaro** [comment]
> 
> 
> @traversaro I don't know the first thing about Windows. If you tell me this is good to go I'll merge it.

For me it is good to go (also thanks to the feedback of @ryanvolz).

**Pull request merged**